<ui version="4.0" >
<comment> Begin CVS Header </comment>
<comment>   $Source: /Volumes/Home/Users/shoops/cvs/copasi_dev/copasi/UI/SensitivitiesWidget.ui,v $ </comment>
<comment>   $Revision: 1.1 $ </comment>
<comment>   $Name:  $ </comment>
<comment>   $Author: pwilly $ </comment>
<comment>   $Date: 2009/04/12 20:05:38 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2008 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., EML Research, gGmbH, University of Heidelberg, </comment>
<comment> and The University of Manchester. </comment>
<comment> All rights reserved. </comment>

 <class>SensitivitiesWidget</class>
 <widget class="TaskWidget" name="SensitivitiesWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>282</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="vboxLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="TextLabel1" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Subtask</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="SubTaskChooser" />
     </item>
     <item row="0" column="2" colspan="2" >
      <spacer name="hSpacer1" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>17</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item rowspan="2" row="1" column="0" colspan="4" >
      <widget class="Line" name="line" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="2" column="1" >
      <widget class="SensWidgetComboBox" name="FunctionChooser" >
       <property name="toolTip" >
        <string>This specifies the set of target values for which&lt;br>the sensitivities are to be calculated.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="TextLabel2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Function</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="2" >
      <spacer name="hSpacer2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>17</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0" >
      <spacer name="hSpacer3" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>97</width>
         <height>24</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1" >
      <widget class="QLineEdit" name="FunctionLineEdit" >
       <property name="minimumSize" >
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>[Please Choose Object.] ---></string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2" >
      <widget class="QToolButton" name="SingleFunctionChooser" >
       <property name="toolTip" >
        <string>If the target value of the sensitivities calculation is a single object &lt;br>the object can be selected by clicking this button.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" >
      <widget class="QLabel" name="TextLabel3" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Variable</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" >
      <widget class="SensWidgetComboBox" name="VariableChooser" >
       <property name="toolTip" >
        <string>This specifies a set of parameters. The sensitivities are calculated with respect to these parameters.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="2" >
      <spacer name="hSpacer4" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>17</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0" >
      <spacer name="hSpacer5" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>97</width>
         <height>24</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1" >
      <widget class="QLineEdit" name="VariableLineEdit" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>[Please Choose Object.] ---></string>
       </property>
      </widget>
     </item>
     <item row="6" column="2" colspan="2" >
      <widget class="QToolButton" name="SingleVariableChooser" />
     </item>
     <item row="7" column="0" >
      <widget class="QLabel" name="TextLabel4" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Second Variable</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" >
      <widget class="SensWidgetComboBox" name="Variable2Chooser" >
       <property name="toolTip" >
        <string>This specifies a second set of parameters. If this is set, second order sensitivities will be calculated.</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2" colspan="2" >
      <spacer name="hSpacer6" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>17</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="0" >
      <spacer name="hSpacer7" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>97</width>
         <height>24</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="1" >
      <widget class="QLineEdit" name="Variable2LineEdit" >
       <property name="minimumSize" >
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>[Please Choose Object.] ---></string>
       </property>
      </widget>
     </item>
     <item row="8" column="2" colspan="2" >
      <widget class="QToolButton" name="SingleVariable2Chooser" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TaskWidget</class>
   <extends>QWidget</extends>
   <header>TaskWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SensWidgetComboBox</class>
   <extends>QComboBox</extends>
   <header>SensWidgetComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>SubTaskChooser</sender>
   <signal>activated(int)</signal>
   <receiver>SensitivitiesWidget</receiver>
   <slot>on_SubTaskChooser_activated(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>144</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel" >
     <x>153</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FunctionChooser</sender>
   <signal>activated(int)</signal>
   <receiver>SensitivitiesWidget</receiver>
   <slot>on_FunctionChooser_activated(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>194</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel" >
     <x>366</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VariableChooser</sender>
   <signal>activated(int)</signal>
   <receiver>SensitivitiesWidget</receiver>
   <slot>on_VariableChooser_activated(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>162</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel" >
     <x>362</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Variable2Chooser</sender>
   <signal>activated(int)</signal>
   <receiver>SensitivitiesWidget</receiver>
   <slot>on_Variable2Chooser_activated(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>253</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel" >
     <x>369</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SingleFunctionChooser</sender>
   <signal>clicked()</signal>
   <receiver>SensitivitiesWidget</receiver>
   <slot>on_SingleFunctionChooser_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>350</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel" >
     <x>366</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SingleVariableChooser</sender>
   <signal>clicked()</signal>
   <receiver>SensitivitiesWidget</receiver>
   <slot>on_SingleVariableChooser_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>347</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel" >
     <x>367</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SingleVariable2Chooser</sender>
   <signal>clicked()</signal>
   <receiver>SensitivitiesWidget</receiver>
   <slot>on_SingleVariable2Chooser_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>343</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel" >
     <x>365</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_SubTaskChooser_activated(int)</slot>
  <slot>on_FunctionChooser_activated(int)</slot>
  <slot>on_VariableChooser_activated(int)</slot>
  <slot>on_Variable2Chooser_activated(int)</slot>
  <slot>on_SingleFunctionChooser_clicked()</slot>
  <slot>on_SingleVariableChooser_clicked()</slot>
  <slot>on_SingleVariable2Chooser_clicked()</slot>
 </slots>
</ui>
